test/base/SpecBase.scala

```diff
@@ -39,14 +39,16 @@ import play.api.test.FakeRequest
 import services.{DateService, RegistrationService}
 import uk.gov.hmrc.auth.core.retrieve.Credentials
 import uk.gov.hmrc.domain.Vrn
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 import uk.gov.hmrc.http.HeaderCarrier
-import viewmodels.checkAnswers._
+import viewmodels.checkAnswers.{IsPlanningFirstEligibleSaleSummary, _}
```

Comment:
if there's an `_` you don't need to add `IsPlanningFirstEligibleSaleSummary` as well